wazo_acceptance/helpers/ha_helper.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2019 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+class HAHelper:
```

Comment:
If we do not add cleanup/logic, I do not see any reason to wrap client into helper

Reply:
Agreed. I removed it.